wgpu-hal/src/vulkan/adapter.rs

```diff
@@ -465,6 +482,32 @@ impl PhysicalDeviceFeatures {
             } else {
                 None
             },
+            mesh_shader: if enabled_extensions.contains(&ext::mesh_shader::NAME) {
+                let needed = requested_features.contains(wgt::Features::MESH_SHADER);
+                Some(
+                    vk::PhysicalDeviceMeshShaderFeaturesEXT::default()
+                        .mesh_shader(needed)
+                        .task_shader(needed)
+                        .multiview_mesh_shader(false),
+                    /*
+                    let multiview = requested_features.contains(wgt::Features::MULTIVIEW);
+
+                    .multiview_mesh_shader(
+                        needed
+                            && multiview
+                            && phd_features.mesh_shader.unwrap().multiview_mesh_shader == 1,
+                    )
+                    */
```

Comment:
Let's fully remove this code, but we can drop a note about multiview_mesh_shader that points to the issue for dealing with multiview mesh shaders